R/sim_contacts.R

```diff
@@ -0,0 +1,82 @@
+#' Simulate contacts for an infectious disease outbreak
+#'
+#' @inheritParams sim_linelist
+#' @param contact_tracing_status_probs A named `numeric` vector with the
+#' probability of each contact tracing status. The names of the vector must
+#' be `"under_followup"`, `"lost_to_followup"`, `"unknown"`. Values of each
+#' contact tracing status must sum to one.
+#'
+#' @return A contacts `<data.frame>`
```

Comment:
Could this have some more detail? E.g. what columns, what is the return suitable for

Reply:
I'm going to leave this `@return` documentation brief for the `sim_*()` functions as listing each column as the contents will be something to update when the output changes (which can easily be forgotten), and the documentation for the `sim_*()` functions is already quite long. IMO the main information the user needs from `@return` is the type/class of the output.